src/test/java/com/savvato/tribeapp/services/PhraseServiceImplTest.java

```diff
@@ -8,16 +8,20 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.system.OutputCaptureExtension;
 import org.springframework.context.annotation.Bean;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 import java.util.Optional;
 
 import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
 
 
-@ExtendWith(SpringExtension.class)
+@ExtendWith({SpringExtension.class, OutputCaptureExtension.class})
```

Comment:
We have other tests which verify the inputs to various calls. Could they also benefit from this OutputCaptureExtension class?

Reply:
Oh shoot, sorry. I missed that. The OutputCaptureExtension was part of how I tested for the log message in my first attempt to verify applyPhraseToUser performed it's actions. I can remove that from this class. The article I found does work with that new slf4j logger you have Sun working on, so if we want to capture any log messages, it will come in handy. Link: [article](https://medium.com/@codebyamir/testing-log-statements-in-spring-boot-ea23d254f06c)

Reply:
No, I was thinking this was a good thing.. I didn't even look into the class, I just assumed it (OutputCaptureExtension.class) was how you accomplished the `verify()` call in your test. Off the top of my head, it looked different, simpler, than other code we have doing similar things in other tests. But now that I look at it, you didn't need to check the values passed to the save() method, so you didn't need the ArgumentCaptor class. Some other test do do that, and as I'm thinking about it now, that is why this test is simpler. I mistakenly attributed that to the OutputCaptureExtension class.